platform/darwin/src/MGLStyle.mm

```diff
@@ -187,7 +187,13 @@ - (MGLSource *)memberOfSources:(MGLSource *)object {
 - (MGLSource *)sourceWithIdentifier:(NSString *)identifier
 {
     auto rawSource = self.mapView.mbglMap->getSource(identifier.UTF8String);
-    return rawSource ? [self sourceFromMBGLSource:rawSource] : nil;
+    MGLSource *source = nil;
```

Comment:
Nit: it isn’t necessary to explicitly set a local variable to `nil` in Objective-C.